pkg/core/function.go

```diff
@@ -550,6 +551,7 @@ func (c *client) UpdateFunction(buildpackBuilder Builder, options UpdateFunction
 		if err != nil {
 			return err
 		}
+		c.bumpNonceAnnotationForRevision(service)
```

Comment:
Isn't that already taken care of by line 525 (previously 524)?

Reply:
same as previous comment